bundesliga_site/views.py

```diff
@@ -48,3 +36,61 @@ def get_auth_token(user):
         print ('UserSocialAuth does not exists!')
     return token
 
+
+@method_decorator(login_required, name='dispatch')
+class SelectEvents(TemplateView, LoginRequiredMixin):
+
+    """ This is the index view. Here we display all the banners that the user
+    has created """
+
+    template_name = 'organizer/select_events.html'
+
+    def _get_event(self):
+        eventbrite = Eventbrite(
+            get_auth_token(self.request.user),
+        )
+        return [
+            event
+            # Status : live, draft, canceled, started, ended, all
+            for event in eventbrite.get(
+                '/users/me/owned_events/?status=live,draft'
+            )['events']
+        ]
+
+    def post(self, *args, **kwargs):
+        # import ipdb; ipdb.set_trace()
+        events = self._get_event()
+        request_body_elements = str(self.request.body).split('event_')
```

Comment:
It looks like you're manually parsing the request body here. There must be a way to have Django do that instead.